platform/linux-generic/test/example/classifier/pktio_env

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(C) 2020 Marvell.
+#
+# Script to setup interfaces used for running application on linux-generic.
+#
+# For linux-generic the default behavior is to create two pcap interfaces
+# and one interface uses udp64.pcap to inject traffic. An output pcap file
+# is generated via second interface.
+#
+# Network set-up
+# +---------+          +-----------+
+# |pcap intf| IF0<---> | Classifier|
+# +---------           +-----------+
+#
+
+PCAP_IN=`find . ${TEST_DIR} $(dirname $0) -name udp64.pcap -print -quit`
+echo "using PCAP in=${PCAP_IN}"
+
+IF0=pcap:in=${PCAP_IN}
+TIME_OUT_VAL=60
+ret=0
+
+if [ "$0" = "$BASH_SOURCE" ]; then
+	echo "Error: Platform specific env file has to be sourced."
+fi
+
+validate_result()
+{
+	CLASSIFIER_LOG="odp_classifier_run.sh.log"
+	sed -i -r 's/.*\r(.*)\r(.*$)/\1\n\2/g' $CLASSIFIER_LOG
+	MAX_PACKETS=$(awk '/queue  pool/{getline; print $1}' $CLASSIFIER_LOG)
+	DEFAULT_MAX_PACKETS=$(awk '/queue  pool/{getline; print $3}' $CLASSIFIER_LOG | \
+		sed 's/^|\(.*\)/\1/')
+	echo "Received packets ${MAX_PACKETS}, default packets ${DEFAULT_MAX_PACKETS}"
+	if [ $MAX_PACKETS != 100 ] || [ $DEFAULT_MAX_PACKETS != 100 ]; then
+		echo "FAIL: Failed to recv packets"
+		ret=1
+	fi
```

Comment:
Instead of parsing the program output, which is prone to breaking, I'd prefer adding a new argument (e.g. -C,--ci-pass) for passing the minimal/exact accepted packet counts per CoS.